build/rocm/dev_build_rocm.py

```diff
@@ -0,0 +1,132 @@
+    try:
+        version = subprocess.check_output("cat /opt/rocm/.info/version | cut -d '-' -f 1", shell=True)
+        return version.decode("utf-8").strip()
+    except subprocess.CalledProcessError as e:
+        print(f"Error fetching ROCm version: {e}")
+        return None
+
+
+def get_rocm_target():
+    try:
+        target_info = subprocess.check_output("rocminfo | grep gfx | head -n 1", shell=True)
+        target = target_info.decode("utf-8").split()[1]
+        return target
+    except subprocess.CalledProcessError as e:
+        print(f"Error fetching ROCm target: {e}")
+        return None
+
+
+def uninstall_existing_packages(packages):
+    for pkg in packages:
+        try:
+            subprocess.run(f"python3 -m pip uninstall {pkg} -y", shell=True, check=True)
+            print(f"Successfully uninstalled {pkg}")
+        except subprocess.CalledProcessError as e:
+            print(f"Failed to uninstall {pkg}: {e}")
+
+
+def clean_dist_directory():
+    try:
+        shutil.rmtree("dist")
+        print("Cleaned dist directory.")
+    except subprocess.CalledProcessError as e:
+        print(f"Failed to clean dist directory: {e}")
+        sys.exit(1)
+
+
+def build_jax_xla(xla_path, rocm_version, rocm_target, use_clang, clang_path):
+    bazel_options = f"--bazel_options=--override_repository=xla={xla_path}" if xla_path else ""
+    clang_option = f"--clang_path={clang_path}" if clang_path else ""
+    build_command = [
+        "python3", "./build/build.py", "--enable_rocm", "--build_gpu_plugin",
+        "--gpu_plugin_rocm_version=60", f"--use_clang={str(use_clang).lower()}",
+        clang_option, f"--rocm_amdgpu_targets={rocm_target}",
+        f"--rocm_path=/opt/rocm-{rocm_version}/", bazel_options
+    ]
+
+    print("Executing build command:")
+    print(" ".join(build_command))
+    
+    try:
+        subprocess.run(" ".join(build_command), check=True)
+        print("Build completed successfully.")
+    except subprocess.CalledProcessError as e:
+        print(f"Build failed: {e}")
+        sys.exit(1)
+
+
+def install_wheel():
+    try:
+        subprocess.run(["python3", "-m", "pip", "install", "dist/*.whl"], check=True)
+        print("Packages installed successfully.")
+    except subprocess.CalledProcessError as e:
+        print(f"Failed to install packages: {e}")
+        sys.exit(1)
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Script to build JAX and XLA on ROCm.")
+    parser.add_argument('--clang_path', type=str, default="",
+                        help="Specify the Clang compiler path")
+    parser.add_argument('--skip_uninstall', action='store_true',
+                        help="Skip uninstall of old versions during package install")
+    parser.add_argument('--use_clang', action='store_true',
+                        help="Use Clang compiler if set")
+    parser.add_argument('--xla_path', type=str, default="",
+                        help="Specify the XLA repository path")
```

Comment:
```suggestion
    parser.add_argument('--clang-path', type=str, default="",
                        help="Specify the Clang compiler path")
    parser.add_argument('--skip-uninstall', action='store_true',
                        help="Skip uninstall of old versions during package install")
    parser.add_argument('--use-clang', action='store_true',
                        help="Use Clang compiler if set")
    parser.add_argument('--xla-path', type=str, default="",
                        help="Specify the XLA repository path")
```

Stick with POSIX like flags https://stackoverflow.com/a/43174117

argparse converts them to _, so you will not have to change your code below.